examples/jsm/exporters/GLTFExporter.js

```diff
@@ -2544,8 +2544,12 @@ class GLTFMaterialsVolumeExtension {
 
```

Comment:
The first line of `writeMaterial` (collapsed above) needs to also check thickness:

```diff
-if ( ! material.isMeshPhysicalMaterial || material.transmission === 0 ) return;
+if ( ! material.isMeshPhysicalMaterial || material.transmission === 0 || material.thickness === 0 ) return;
```

Otherwise we may be always adding KHR_materials_volume to any material that uses transmission, which isn't necessary and may reduce compatibility. The extension is effectively disabled if `thicknessFactor=0`.

Reply:
Sounds good. This change has been added to the branch.